src/index.js

```diff
@@ -50,7 +50,7 @@ function prometheusMetricsExporterWrapper(options = {}) {
 
 function httpMetricMiddlewareWrapper(options = {}) {
     // setup metrics.
-    const labelNames = ['method', 'uri', 'code'];
+    const labelNames = ['status_code', 'method', 'path'];
```

Comment:
The code passes values as `(method, path, status)` though so this would be wrong. Can you explain what you're trying to do?

Reply:
`Hi @jonbretman,

I am using express.js for the other application. As per my requirement, I need to label names as code ->status_code, method -> method, URI -> path.  I am tracking all the Prometheus metrics with the default labels. I found this library contains some changes. If you accept my request,  I can raise a pull request again with the little modifications.

Thanks.